packages/block-library/src/preformatted/transforms.js

```diff
@@ -35,6 +35,11 @@ const transforms = {
 			transform: ( attributes ) =>
 				createBlock( 'core/paragraph', attributes ),
 		},
+		{
+			type: 'block',
+			blocks: [ 'core/code' ],
+			transform: ( attributes ) => createBlock( 'core/code', attributes ),
```

Comment:
I noticed that the "preformatted" content attribute has this property on it `"__unstablePreserveWhiteSpace": true`
I wonder if that means that we should sanitize the content when transforming cc @ellatrix 
Also, do we want to support the opposite direction too?

Reply:
> Also, do we want to support the opposite direction too?

Yes, but this is already implemented, see my gif in the Screenshot section.

Reply:
I think it's fine, because a `textarea` doesn't need anything special to preserve whitespace.

Reply:
And I believe we use characters for line breaks when `__unstablePreserveWhiteSpace` is enabled, so I don't think there's any HTML to strip. Worth noting that `pre` can contain HTML though. Can have bold, links etc.